CMakeLists.txt

```diff
@@ -24,6 +23,15 @@ option(FORCE_CXX11
     OFF)
 enable_testing()
 
+# Python can not be binged with mlpack-master on windows yet
```

Comment:
Should this be ``can not be build with mlpack``or ``Currently Python bindings aren't supported on windows.``, which might be possible if we use anaconda, I'll check that over the next days.